ark/phenotyping/som_utils.py

```diff
@@ -129,21 +126,19 @@ def create_fov_pixel_data(fov, channels, img_data, seg_labels,
     return pixel_mat, pixel_mat_subset
 
 
-def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
+def create_pixel_matrix(fovs, base_dir, tiff_dir, seg_dir,
                         pre_dir='pixel_mat_preprocessed',
                         sub_dir='pixel_mat_subsetted', is_mibitiff=False,
-                        blur_factor=2, subset_proportion=0.1, seed=42):
-    """Preprocess the images for FlowSOM clustering and creates a pixel-level matrix
+                        blur_factor=2, subset_proportion=0.1, seed=None):
```

Comment:
Is there a reason some of the functions are initialized with None and some with 42?